src/components/Book.js

```diff
@@ -3,40 +3,66 @@ import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import Button from './Button';
 import { removeBook, removeAsyncBook } from '../redux/books/booksSlice';
+import './styles/book.css';
 
-function Book({ id, title, author }) {
+function Book({
+  id, title, author, category,
+}) {
   const dispatch = useDispatch();
   const handelRemoveBook = () => {
     dispatch(removeAsyncBook(id));
     dispatch(removeBook(id));
   };
   return (
-    <>
-      <div>
-        <ul>
+    <div className="Lesson-Panel">
+      <p className="school-of">
+        {category}
+        {' '}
+      </p>
+      <h4 className="title">{title}</h4>
+      <p className="suzanne-collins">{author}</p>
+      <div className="flex-spac-btwn">
+        <div className="buttons-contanier flex">
+          <Button className="comments-btn" value="Comment" type="button" />
+          <hr />
+          <Button className="remove-btn" type="button" onClick={handelRemoveBook} value="Remove" />
+          <hr />
+          <Button className="edit-btn" value="Edit" type="button" />
```

Comment:
- [ ] Kindly remember to add these grey vertical lines as well in your design so that your design can match the zeplin design

![image](https://github.com/naji-97/sec-book-store/assets/63473584/c917dd6d-b58d-4492-a1b2-5500ae2b6546)